src/azure-firewall/HISTORY.rst

```diff
@@ -2,6 +2,24 @@
 
 Release History
 ===============
+0.9.1
+++++++
+* Migrate to Track2 SDK.
+
+0.9.0
+++++++
+* az network firewall policy rule-collection-group collection add-filter-collection: Add parameter 'web-categories'
+* az network firewall policy rule-collection-group collection rule add: Add parameter 'web-categories'
+* Intrusion Signature Override Arguments->IDPS Signature Override Arguments
```

Comment:
Could you check with @msyyc to make this change log more consistent with others?

Reply:
All checked